src/priority_q.py

```diff
@@ -0,0 +1,32 @@
+from bin_heap import BinHeap
+
+
+class PQueue(object):
+    """Priority Queue object."""
+
+    def __init__(self):
```

Comment:
Since your `BinHeap` class can be constructed from an iterable passed in to the class constructor, should PQueue also be able to take an iterable on construction? Not necessary for grades, just worth considering.